### ISSUE TITLE
Suggestions / feature requests

### ISSUE THREAD

**Issue by Not-Sure-Whats-Going-On** [jasnoo/nature-week#5]
This is great idea Jasmine and nicely implemented - bookmarked!
I have three suggestions based on my first time using it ...  

1. I am typing this on Sunday June 25th and the results are shown as "Seen between Jun 25, 2023 - today."  Is a week defined as Sunday to Saturday?  Would it be possible to make it rolling - i.e. the previous 7 days?

2. At first I was confused about what I should enter in the Your Location box - GPS location, Latitude & Longitude?  I only discovered by accident that it should be the iNaturalist "Place" name - maybe add an onMouseOver explanation to improve usability?

3. Would it be possible to add choices for other groups like insects too?

Thanks.
John

**Comment by jasnoo** [comment]
Hi John, 

Thanks for your feedback! I've fixed the date bug - it was intended to be a rolling window so I corrected the behavior. For other options, I've added insects in for now - if there are others you are interested in, I can consider them as well. Depending on how many filters I end up including I may change the UI functionality here. But for now I like having them as simple buttons given the minimal options. Finally for the location feedback, I agree that it's kind of confusing here. I am closing this without changes for now, but I will work to improve this hopefully adding more flexibility around what can be included here (for instance, allowing a user to detect location or inputting lat/long), otherwise certainly adding a tool tip for explanation.

I'm really glad you liked this project :)

Jasmine

**Comment by jasnoo** [comment]
Hi John,

Thanks for your feedback! I've fixed the date bug - it was intended to be a rolling window so I corrected the behavior. For other options, I've added insects in for now - if there are others you are interested in, I can consider them as well. Depending on how many filters I end up including I may change the UI functionality here. But for now I like having them as simple buttons given the minimal options. Finally for the location feedback, I agree that it's kind of confusing here. I am closing this without changes for now, but I will work to improve this hopefully adding more flexibility around what can be included here (for instance, allowing a user to detect location or inputting lat/long), otherwise certainly adding a tool tip for explanation.

I'm really glad you liked this project :)

Jasmine
